build.gradle

```diff
@@ -44,6 +47,13 @@ allprojects {
   repositories {
     jcenter()
     maven { url "https://jitpack.io" }
-    maven { url "https://maven.google.com" }
+    maven {
+      url "http://skedgo.bintray.com/tripkit-android"
+      credentials {
+        username BINTRAY_USER
+        password BINTRAY_KEY
+      }
+    }
```

Comment:
We don't use this bintray maven anymore. Can you please remove?

Reply:
yes